### DISCUSSION TITLE
Is there any way to revert to simple password. Organisation has been removed.

### DISCUSSION THREAD

**Discussion by queenoftheberg** [community/community#118990] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I created an organisation because another person was going to work with me on some projects.
However, I landed up doing all the work, since I am the more experienced web designer.
2FA seems to be necessary for people with organisations.
I have since moved my repositories back to my own name and deleted the organisation.

I DISABLED 2FA, but now I am getting reminders to activate 2FA . . .

I seriously dislike 2FA, because access to my repositories seems out of my control.
A list of Codes, which are supposed to activate / log in to my account, no longer works.
Our laptop crashed and a borrowed laptop has all my stuff on it.
I saved the list of codes in a spreadsheet, but the file of codes is not on this laptop. Is that why they no longer work?

An authorisation app seemed to be able to log me in, but I have no idea how it works, or how it decides if I'm the right person to give access to. 

Now I just want to be an ordinary user, without an organisation.  I understand the risks in big business, but I'm just one person.  I put all my FreeCodeCamp profile projects onto Github as well.  If I must remove them to 'downsize' I can do that too.

Please can I have my password access back. ... I can change my password every 2nd week if necessary.  At least then I get to decide how/who gets access to my repositories.

**Comment by ettaboyle** [comment]
The best (and only) place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
